<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="elevator_frame">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>301</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="elv_help_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f7_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_fire_btn">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>200</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f4_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f5_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f6_btn">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>270</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_open_btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_close_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f3_btn">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>340</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f2_btn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elv_f1_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator_speaker">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>Speaker</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="elevator_display">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="floor_frame">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="floor_down_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="floor_up_btn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="floor_display">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="floorNum_box">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="ECS_frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="elv1_door">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Door</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elv1_overload">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elv2_door">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Door</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elv3_door">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Door</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elv2_overload">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="elv3_overload">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator1_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>65</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator1_label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>115</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator1_label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>165</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="ECS_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Elevator Control System</string>
     </property>
    </widget>
    <widget class="QLabel" name="elv1_floorNum">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>62</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="elv3_floorNum">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>162</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="elv2_floorNum">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>112</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="elvNum_box">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="BSS_frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="BSS_fire">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BSS_powerOut">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BSS_submit_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="BSS_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Building Safety Service</string>
     </property>
    </widget>
    <widget class="QLabel" name="BSS_help_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="building_response">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>135</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Building</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="passenger_response">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Passenger</string>
     </property>
    </widget>
    <widget class="QLabel" name="response_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Response</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="disconnect_response">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>185</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="floor_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>65</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="elevator_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>25</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elevator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElevator_Simulator">
    <property name="title">
     <string>Elevator Simulator</string>
    </property>
   </widget>
   <addaction name="menuElevator_Simulator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
